### ISSUE TITLE
[A11y] Improve accessibility for the Author block

### ISSUE THREAD

**Issue by jlemay-godaddy** [godaddy-wordpress/coblocks#1963]
**Accessibility problem:**
No focus indicator when landing on the test author button

Requested in #1927

**Comment by olafleur-godaddy** [comment]
Closing, as there is a focus indicator (e.g. dotted line) when landing on the author button.
<img width="629" alt="Screen Shot 2021-10-06 at 11 13 56" src="https://user-images.githubusercontent.com/85255688/136232347-20cb8598-9b5d-4ac1-97ba-3455f6c441ac.png">
